perun-scim/src/main/java/cz/metacentrum/perun/scim/api/entities/UserSCIM.java

```diff
@@ -41,14 +53,19 @@ public UserSCIM(Long id, Long externalId, Meta meta, List<String> schemas, Strin
 	}
 
 	public UserSCIM(Long id, Long externalId, Meta meta) {
-		super(id, externalId, meta);
+		resource = new Resource();
+		resource.setId(id);
+		resource.setExternalId(externalId);
+		resource.setMeta(meta);
 	}
 
 	public UserSCIM() {
+		resource = new Resource();
 	}
 
+	@JsonIgnore
```

Comment:
We want to ignore just resource and its getters and setters, so please remove JsonIgnore annotation for getSchemas() and also for getEmails().